README.md

```diff
@@ -7,7 +7,7 @@ Server-side tools can be found [here](https://github.com/MEPP-team/UDV-server).
 
 ## Demo
 Online demos (alas only visible on the [Lyon1](https://sciences.univ-lyon1.fr/)/[INSA-Lyon](https://www.insa-lyon.fr/en/) campus):
- - [UDV](http://rict.liris.cnrs.fr/UDVDemo-2/UDV/UDV-Core/)
+ - [UDV](http://rict.liris.cnrs.fr/UDV/UDV-Core/)
```

Comment:
Broken link and I think that the demos are visible outside from the campus now

Reply:
Yes, I fixed it and removed the "only visible on..."